README.md

```diff
@@ -30,27 +30,17 @@ curl http://my-url
 All without needing to worry about writing an HTTP server or request
 handling logic.
 
-# Features
+## Features
 
 *   Spin up a local development server for quick testing with little extra code
 *   Invoke a function in response to a request
 *   Automatically unmarshal events conforming to the
     [CloudEvents](https://cloudevents.io/) spec
 *   Portable between serverless platforms
 
-# Quickstart: Hello, World on your local machine
+## Quickstart: Hello, World on your local machine
 
-1. Make sure you have Go 1.11+ installed with:
-	```
-	go version
-	```
-	The output should be Go 1.11 or higher.
-
-1. Create the necessary directories.
-	```sh
-	mkdir -p hello/cmd
-	cd hello
-	```
+1. Install Go 1.11+, [Docker](https://store.docker.com/search?type=edition&offering=community), and the [`pack` tool](https://buildpacks.io/docs/install-pack/).
```

Comment:
This instruction changes the quickstart to something more complicated and not as quick.

I'm not sure Function developers understand or want to use Docker or buildpacks. Can we provide this tutorial as an alternative to the one we had before?

This PR was meant to fix the README. It seems like bad practice to require a lot of dependencies just to get started with Functions.

Reply:
Given that the alternative is the not-so-quick "go write an entire function server that negates the benefit of the framework", I'd argue that this is a better and quicker option. I think this clarifies and cleans up the readme, since it doesn't force the user to write unnecessary code that will never be used in production.

Reply:
I see there are trade-offs to both approaches:

- One way faster but not like production
- One is slower but matches production

The point about code not seen in production is huge and should be emphasized in the README.

For some languages like Node, I'd expect there to not be the developer cycle of build then run (just Run), but for Go, I guess the extra seconds waiting for building a container is fine.

---

Can we write this down? i.e. feature:

- Build your Function in the same container environment seen by Cloud Functions using [Buildpacks](https://github.com/GoogleCloudPlatform/buildpacks).

Reply:
I'd argue that the buildpacks way is faster, too, given that you don't have to write and maintain the extra code, but I'll add the line.

Reply:
I think I'd prefer to show the code here. We can link to the buildpack docs and say, "if you don't want to write any code go [here]" (not exactly that, but you get the gist).